image/rhel/konflux.Dockerfile

```diff
@@ -52,6 +60,44 @@ COPY .konflux/rocksdb ./
 RUN PORTABLE=1 TRY_SSE_ETC=0 TRY_SSE42="-msse4.2" TRY_PCLMUL="-mpclmul" CXXFLAGS="-fPIC" make --jobs=6 static_lib
 
 
+FROM scratch AS go-builder
+
+ARG GO_BUILDER_STAGE_PATH
+COPY --from=rpm-installer "$GO_BUILDER_STAGE_PATH" /
+
+COPY --from=rocksdb-builder /staging/librocksdb.a /lib/rocksdb/librocksdb.a
+COPY --from=rocksdb-builder /staging/include /lib/rocksdb/include
+
+WORKDIR /go/src/github.com/stackrox/rox/app
+
+COPY . .
+
+# TODO: remove me
```

Comment:
Possible workaround: https://github.com/stackrox/stackrox/blob/master/scripts/konflux/fail-build-if-git-is-dirty.sh#L17-L19.

Reply:
I felt that git-restoring any `package-lock.json` files in Konflux context is a bit too custom for `fail-build-if-git-is-dirty.sh`, and besides it may break UI build if `fail-build-if-git-is-dirty.sh` is called at UI build stage (no case right now but who knows).

My suggestion would be to just remove this comment and this `git diff`. I am just not sure if that's really Konflux changing it and what's the nature of the changes. I wanted to look at logs for this. Unfortunately, logs for the latest build are gone at the moment.